upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.7.yaml.template

```diff
@@ -216,6 +217,8 @@ spec:
               value: "{{- or .Networking.Calico.PrometheusProcessMetricsEnabled "true" }}"
             - name: FELIX_HEALTHENABLED
               value: "true"
+            - name: FELIX_IPINIPMTU
```

Comment:
What is the default if not set? ... Also, the [docs](https://docs.projectcalico.org/v3.0/usage/configuration/mtu) indicate encapsulation and cloud provider makes a difference

Reply:
The default is 8918 if not set. Users can set it to the correct option based on encapsulation and cloud provider.